gum/backend-windows/gumprocess-windows.c

```diff
@@ -34,11 +34,25 @@ gum_process_is_debugger_attached (void)
   return IsDebuggerPresent ();
 }
 
+#if defined (HAVE_I386) && GLIB_SIZEOF_VOID_P == 4
```

Comment:
We usually have an empty line on each side of the `#if` blocks elsewhere, so we should follow that here as well.